usefulUtilities/voteApp/appResources/voteApp.js

```diff
@@ -0,0 +1,1022 @@
+"use strict";
+
+//  
+//  Vote App | voteApp.js
+//
+//  Created by Robin Wilson 2018-11-12
+//
+//  Using Example Vue App Created by Milad Nazeri on 2018-10-11
+//  Copyright 2016 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+/* globals Tablet, Script, HMD, Controller, AccountServices, Menu */
+
+(function () { // BEGIN LOCAL_SCOPE
+
+    // Modules
+    var AppUi = Script.require('appUi'),
```

Comment:
Coding standards - declare/define variables as close as reasonable to where they're first used